styles.css

```diff
@@ -21,16 +21,16 @@ button {
   height: 60px;
   background-color: gray;
   color: white;
-  border: 2px solid black;
+  border: 3px solid #5a5a5a;
```

Comment:
Whoops, I think I changed the borders on the text input fields to go back down to 2px -- I think that's what they use in the comp. MERGE CONFLICT AHEAD